node/test/python/aws-lambda-sdk/benchmark/README.md

```diff
@@ -0,0 +1,115 @@
+# Benchmarks
+
+## Tested use cases
+
+### Success function ([`success`](../../../../../python/packages/aws-lambda-sdk/tests/fixtures/lambdas/success.py))
+
+No-op success function. Handler takes callback and immediately invokes it with dummy payload
+
+## Benchmark variants
+
+### `bare`
+
+No instrumentation (bare run with no extension layers onboard).
+
+As there are no extensions loaded, there's no extension duration overhead introduced, and benchmark will naturally report them with zeros.
+
+What we get is metrics from AWS report, that confirm on initialization duration, invocation duration, billed invocation duration and max memory used.
+
+### `internal`
+
+Internal extension which calculates and logs generated traces payload to the console
+
+## Reported metrics
+
+Each benchmark variant is run against 5 lambda instances.
+
+Therefore for each metric outlined below, we get 5 numbers. Durations returned in result report are averages of those 5 values.
+
+Internally measured durations are written by the extensions to the `stdout`, and are read (together with invocation reports as written by AWS) from CloudWatch logs.
+
+### Observed AWS Lambda execution lifecycle phases
+
+#### Initialization (`init`)
+
+Lambda instance initialization
+
+##### Observed metrics
+
+###### `init:internal`
+
+Duration of the internal extension initialization logic, measured internally in internal extension. It's a time measured from start of the internal extension logic, until it returns for further evaluation by AWS runtime engine.
+
+###### `init:total`
+
+Total initialization time as reported by AWS (with `initDuration` metric provided in report of the first invocation)
+
+#### Invocation
+
+Benchmarks observe first invocation (`first`) (one that in regular circumstances immediately follows initialization), and the the second (`following`).
+
+Two first invocations are observed separately, as data shows that durations of first invocation are always worse than of the second, and that for the following (third, fourth etc.) invocations numbers are similar as for the second.
+_This difference could be attributed to how Node.js internal optimization works, where paths which were already taken usually run faster (it's to be confirmed whether we see similar differences with other runtimes)._
```

Comment:
This part talks about nodejs internal performance. Is that still true for this benchmark given it is written in node or should this be removed? 🤔